mtags/src/main/scala-2/scala/meta/internal/pc/AutoImportsProvider.scala

```diff
@@ -21,9 +21,12 @@ final class AutoImportsProvider(
       filename = params.uri().toString(),
       cursor = Some(params.offset())
     )
+    typeCheck(unit)
```

Comment:
looks like it should be removed

Reply:
This is introduced by https://github.com/scalameta/metals/pull/4223

Reply:
:thinking: cross test pass for scala2 without this line.
@tgodzik do you remember why it was needed?

Reply:
och no, that might have been by mistake. Should be ok to remove it.

Reply:
Ah, ok then let's remove this line :+1:

Reply:
Removed